### PR TITLE
Add travis integration

### PR DESCRIPTION
**Pull request by ChadKillingsworth**
I'd like to add npm/grunt/gulp integrations and have PRs auto-run any tests. The first step in this is to enable travis-ci testing for the existing unit tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 The Polymer Renamer
 ===================
+
+[!Travis Build Status](https://travis-ci.org/PolymerLabs/PolymerRenamer.svg?branch=master)
```
Comment (robliao) [inline README.md]: For consistency with other Polymer projects, let's use this as the travis link:

```
[![Build Status](https://travis-ci.org/PolymerLabs/PolymerRenamer.svg?branch=master)](https://travis-ci.org/PolymerLabs/PolymerRenamer)
```

Comment (robliao) [inline README.md]: Thanks for the PR! Left a comment and then I'll merge the PR into master.

Comment (ChadKillingsworth) [inline README.md]: Done!

**Comment by robliao** [comment]
lgtm. Thanks for the PR!

**Comment by robliao** [comment]
Reopening Pull Request

**Pull request merged**